Modules/TOF/include/TOF/TaskRaw.h

```diff
@@ -68,6 +68,8 @@ class RawDataDecoder final : public DecoderBase
   void setTimeWindowMin(std::string min) { mTimeMin = atoi(min.c_str()); }
   void setTimeWindowMax(std::string max) { mTimeMax = atoi(max.c_str()); }
   void setNoiseThreshold(std::string thresholdnoise) { mNoiseThreshold = atof(thresholdnoise.c_str()); }
+  void setDebugCrateMultiplicity(bool debug) { mDebugCrateMultiplicity = debug; }
+  bool isDebugCrateMultiplicity() { return mDebugCrateMultiplicity; }
```

Comment:
this could be changed to ` const bool& isDebugCrateMultiplicity() const { return mDebugCrateMultiplicity; }`